test/integration/camera_sensor_background_from_scene.cc

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include <ignition/transport/Node.hh>
+#include <ignition/utilities/ExtraTestMacros.hh>
+
+#include "ignition/gazebo/Server.hh"
+#include "ignition/gazebo/SystemLoader.hh"
+#include "ignition/gazebo/Util.hh"
+#include "ignition/gazebo/test_config.hh"
```

Comment:
We have to convert everything to `gz`, this can be done in a new commit with this PR in the name

```
/home/jenkins/workspace/ignition_gazebo-ci-pr_any-ubuntu_auto-amd64/ign-gazebo/test/integration/camera_sensor_background_from_scene.cc:23:10: fatal error: ignition/utilities/ExtraTestMacros.hh: No such file or directory
   23 | #include <ignition/utilities/ExtraTestMacros.hh>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
15b6d75938579def2e6fef19587da8fbac1b9dd3. This I had to change to `gz/utils/`.